examples/templates/devcontainer-kubernetes/main.tf

```diff
@@ -1,19 +1,24 @@
 terraform {
   required_providers {
     coder = {
-      source = "coder/coder"
+      source  = "coder/coder"
+      version = "~> 1.0.0"
     }
-    kubernetes = {
-      source = "hashicorp/kubernetes"
+    docker = {
+      source = "kreuzwerker/docker"
```

Comment:
Maybe add a comment on why the kubernetes template uses `kreuzwerker/docker`

Reply:
oops, bad copy-paste